src/style.css

```diff
@@ -1 +1,125 @@
 /* add styles here */
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 600;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 8px;
+  padding: 0;
+  font-size: 16px;
+  font-family: Avenir, sans-serif;
```

Comment:
Remember to use fallback fonts. In case the main font doesn't load, the browser can use the next available font. For example, 'font-family: Avenir, Arial, sans-serif;'